test/integration/sanity/DeletedReps.e2e.test.js

```diff
@@ -1,46 +1,20 @@
 // <reference types="Cypress" />
 describe('Previewing a file with deleted representations', () => {
-    const token = Cypress.env('ACCESS_TOKEN');
-
     const fileIdDoc = Cypress.env('FILE_ID_DOC');
     const fileIdPresentation = Cypress.env('FILE_ID_PRESENTATION');
-
-    const REPS_ERROR = 'error_deleted_reps';
-
-    const helpers = {
-        checkDeletedRepError: () => {
-            cy.window().then(win => {
-                win.preview.addListener('preview_error', data => {
-                    cy.expect(data.error.code).to.equal(REPS_ERROR);
-                });
-            });
-        },
-    };
+    const token = Cypress.env('ACCESS_TOKEN');
 
     beforeEach(() => {
         cy.server();
-        // Make the representation content call to always return a 202 with no body
-
-        // Mocking requests for non original reps
-        cy.route({
-            method: 'GET',
-            url: '**/internal_files/**',
-            status: 202,
-            response: {},
-        });
-
-        // Mocking requests for original reps
-        cy.route({
-            method: 'GET',
-            url: '**/content?**',
-            status: 202,
-            response: {},
-        });
 
         cy.visit('/', {
             onBeforeLoad(win) {
                 // Workaround for fetch detection in cypress mocking. https://github.com/cypress-io/cypress/issues/95
-                delete win.fetch; // eslint-disable-line no-param-reassign
+                cy.stub(win, 'fetch').resolves({
```

Comment:
I confirmed this change still triggers the expected error both on this branch and on master.